Source/ocdm/open_cdm_impl.h

```diff
@@ -367,22 +368,22 @@ struct OpenCDMSession {
         }
 
         BEGIN_INTERFACE_MAP(Sink)
-        INTERFACE_ENTRY(OCDM::ISession::ICallback)
+        INTERFACE_ENTRY(Exchange::ISession::ICallback)
         END_INTERFACE_MAP
 
     private:
         OpenCDMSession& _parent;
     };
 
-    class DataExchange : public OCDM::DataExchange {
+    class DataExchange : public Exchange::DataExchange {
     private:
         DataExchange() = delete;
         DataExchange(const DataExchange&) = delete;
         DataExchange& operator=(DataExchange&) = delete;
 
     public:
         DataExchange(const string& bufferName)
-            : OCDM::DataExchange(bufferName)
+            : WPEFramework::Exchange::DataExchange(bufferName)
```

Comment:
Eh why now suddenly also include WPEFramework namespace? See my comment in the Interfaces Pullrequest, we try to minimize the inclusion of that namespace. Please remove it in the pull request where possible